lib/models/balenaos-contract.ts

```diff
@@ -44,6 +44,9 @@ const BalenaOS: Contract = {
 			`Wait for writing of {{name}} to complete.`,
 			`{{{deviceType.partials.bootDevice}}} to boot the device.`,
 		],
+		radxaFlash: [
```

Comment:
Can you also add a test case at
See: https://github.com/balena-io/balena-sdk/blob/master/tests/integration/models/device-type.spec.ts#L112-L128

so that we can see how this works for such a device type?

Reply:
I added one now but it will be necessary for https://github.com/balena-io/contracts/pull/363 to be merged first, right @vipulgupta2048 ?

Reply:
Yeah, we need to change the implementation of a method first as well for these instructions to be used. Making a commit now